### PR TITLE
create timestamp view (#219)

### PR DESCRIPTION
**Pull request by Petros9**

### CODE REVIEW

**Review by michal367 — APPROVED** [review]
Great

File: src/lecturer/components/timestampView/TimestampView.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { makeStyles, useTheme } from "@material-ui/core";
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { useBackEnd, useBackEndSocket } from "../../services/BackEndService";
+import { StoreContext } from "../../services/StoreService";
+import { useHistory } from "react-router-dom";
+import { useLocation } from "react-router-dom";
+import StudentsQuestionListView from "../studentsQuestionView/StudentsQuestionListView";
+import {TimestampTable} from "./TimestampTable";
```
Comment (ostatni5) [inline src/lecturer/components/timestampView/TimestampView.tsx]: remove unused imports

**Review by ostatni5 — APPROVED** [review]
Remove unnecessary code and you are good to go.

a

**Pull request merged**